ipyparallel/cluster/shellcmd.py

```diff
@@ -0,0 +1,860 @@
+#!/usr/bin/env python
+"""
+Shell helper application for OS independent shell commands
+
+Currently, the following command are supported:
+* start:   starts a process into background and returns its pid
+* running: check if a process with a given pid is running
+* kill:    kill a process with a given pid
+* mkdir:   creates directory recursively (no error if directory already exists)
+* rmdir:   remove directory recursively
+* exists:  checks if a file/directory exists
+* remove:  removes a file
+"""
+import base64
+import enum
+import inspect
+import json
+import os
+import re
+import shlex
+import sys
+import time
+from datetime import datetime
+from random import randint
+from subprocess import CalledProcessError, Popen, TimeoutExpired, check_output
+from tempfile import NamedTemporaryFile
+
+
+class Platform(enum.Enum):
+    Unknown = (0,)
+    Linux = (1,)
+    Windows = (2,)
```

Comment:
presumably these are meant to be integers

Reply:
Those are integeres but in a strange syntax. This was somehow caused by pre-commit. fixed by 556eeb8e3442461f3ba088e0f92c4db9d12f0448